packages/dapp/components/lib/utils.ts

```diff
@@ -28,10 +27,10 @@ export async function performTransaction(transaction: Promise<ContractTransactio
   // return false;
 }
 
-export async function erc1155BalanceOf(addr: string, erc1155UbiquityCtr: ERC1155Ubiquity): Promise<BigNumber> {
+export async function erc1155BalanceOf(addr: string, erc1155UbiquityCtr: Contract): Promise<BigNumber> {
   const treasuryIds = await erc1155UbiquityCtr.holderTokens(addr);
 
-  const balanceOfs = treasuryIds.map((id) => {
+  const balanceOfs = treasuryIds.map((id: any) => {
```

Comment:
We should not be using `any` perhaps try using a more specific type like `number` or `string`. If none work then I would fall back to `unknown`.

I assumed that our commit hook would throw an error for this!

Reply:
https://github.com/ubiquity/ubiquity-dollar/pull/249/commits/76d0b63e397ad3c7ef849d9f5c31f946ae6603c9